recipes-kernel/linux/linux-imx_5.10.bb

```diff
@@ -13,20 +13,20 @@ i.MX Family Reference Boards. It includes support for many IPs such as GPU, VPU
 require recipes-kernel/linux/linux-imx.inc
 
 LICENSE = "GPLv2"
-LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
 
 DEPENDS += "lzop-native bc-native"
 
-SRCBRANCH = "imx_5.4.70_2.3.0"
-LOCALVERSION = "-2.3.2"
-SRCREV = "5f2220e87dd8a8fb86f524ea7945ab1d4155bc73"
+SRCBRANCH = "lf-5.10.y"
+LOCALVERSION = "-2.0.0"
+SRCREV = "ef3f2cfc6010c13feb40cfb7fd7490832cf86f45"
```

Comment:
It is a bit odd to see that version goes backwards here: from `2.3.0` back to `2.0.0`.

I known that it is rather a "cosmetic" point, but I hope that people do not rely much on the output of `uname` in this case.

Reply:
Would `-imx-5.10.35-2.0.0` be better? What about `linux-fslc-imx`, it currently follows the same scheme? Should I update it like this?

`linux-fslc-imx_5.4.bb:LOCALVERSION = "-fslc-imx-5.4.70-2.3.0"`

Reply:
@thochstein 

I would not mind doing this change.

I can include it in the kernel PR I'm currently working on for `linux-fslc-imx`, and change _LOCALVERSION_ from `-imx-5.4.70-2.3.0` to `-fslc-imx-5.4.70-2.3.0` with consecutive change to `-imx-5.10.35-2.0.0` once kernel branch would be created.

@otavio what do you think about this?

-- andrey

Reply:
`5.10.35-2.0.0` is shorten.

Reply:
So @zandrey should update fslc version to `-fslc-5.10.35-2.0.0`?

Reply:
I believe `-5.4.70-2.3.0` is enough as this is the BSP version it is based on.

Reply:
@otavio OK, no problem!

I'll push one more commit on the upcoming kernel PR to change _LOCALVERSION_ to `-5.4.70-2.3.0`

@thochstein I believe you would then use the `-5.10.35-2.0.0` for _LOCALVERSION_, right? In this case we would have both recipes aligned.

-- andrey

Reply:
Done, please see a1ff57ba ("linux-fslc-imx: sanitize local version") in #820